.changeset/spotty-baboons-rule.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
maybe merge the 2 changesets, they are related

Reply:
would having the 2 changesets merged have some benefit? or put differently, does having 2 separate changesets have downsides?

Reply:
Up to you